target/linux/mediatek/filogic/base-files/etc/board.d/01_leds

```diff
@@ -10,9 +10,9 @@ cudy,wr3000-v1)
 	ucidef_set_led_netdev "wan" "wan" "blue:wan" "wan"
 	;;
 mercusys,mr90x-v1)
-	ucidef_set_led_netdev "lan0" "lan0" "green:lan0" "lan0" "link tx rx"
-	ucidef_set_led_netdev "lan1" "lan2" "green:lan1" "lan1" "link tx rx"
-	ucidef_set_led_netdev "lan2" "lan2" "green:lan2" "lan2" "link tx rx"
+	ucidef_set_led_netdev "wan/lan" "wan/lan" "green:lan-0" "lan0" "link tx rx"
```

Comment:
I haven't seen this (`wan/lan`) used elsewhere.
But TBH, `uci_def_set_led_netdev` is a standardless mess... (see `grep -r set_led_netdev target/linux/ramips/`)

Reply:
`wan/lan` - this is the name of the port on the router case. If we touch the port label in dts we need to update `02_network`. It would also be nice to make migration scripts. A tiny change grows larger and larger. So I will revert `lan0 `back.